.github/actions/docker-build-and-push-cuda/action.yaml

```
@@ -0,0 +1,121 @@
+name: docker-build-and-push
+description: ""
+
+inputs:
+  platform:
+    description: ""
+    required: true
+  bake-target:
+    description: ""
+    required: true
+  build-args:
+    description: ""
+    required: false
```

```suggestion
inputs:
  platform:
    description: "Target platform."
    required: true
  target-image:
    description: "Target docker image name in the registry."
    required: true
  build-args:
    description: "Additional build args."
    required: false
```

Consider updating the description(s).

---

Additionally the `bake-target` is only used for image name, maybe consider renaming it to `image-name`?

E.g. 

``` yaml
 images: ghcr.io/${{ github.repository_owner }}/${{ inputs.bake-target }}
```

As we do not use it for

``` yaml
bake-target: docker-metadata-action-universe-sensing-perception-devel-cuda
```

---

It's good suggestion but we also need to modify `docker-build-and-push/action.yaml` for consistency.
https://github.com/autowarefoundation/autoware/blob/main/.github/actions/docker-build-and-push/action.yaml#L11
I'll make another PR after merging this.